src/main/java/com/endava/soa/pageObject/AnouncePlacingMobilePhonePage.java

```diff
@@ -0,0 +1,297 @@
+        this.rezolutieCameraDropdown = rezolutieCameraDropdown;
+    }
+
+    public WebElement getSenzorDeAmprentaCheckbox() {
+        return senzorDeAmprentaCheckbox;
+    }
+
+    public void setSenzorDeAmprentaCheckbox(WebElement senzorDeAmprentaCheckbox) {
+        this.senzorDeAmprentaCheckbox = senzorDeAmprentaCheckbox;
+    }
+
+    public WebElement getDeblocareFacialaCheckbox() {
+        return deblocareFacialaCheckbox;
+    }
+
+    public void setDeblocareFacialaCheckbox(WebElement deblocareFacialaCheckbox) {
+        this.deblocareFacialaCheckbox = deblocareFacialaCheckbox;
+    }
+
+    public WebElement getNfcCheckbox() {
+        return nfcCheckbox;
+    }
+
+    public void setNfcCheckbox(WebElement nfcCheckbox) {
+        this.nfcCheckbox = nfcCheckbox;
+    }
+
+    public WebElement getUsbTypeCheckbox() {
+        return usbTypeCheckbox;
+    }
+
+    public void setUsbTypeCheckbox(WebElement usbTypeCheckbox) {
+        this.usbTypeCheckbox = usbTypeCheckbox;
+    }
+
+    public WebElement getDualSimCheckbox() {
+        return dualSimCheckbox;
+    }
+
+    public void setDualSimCheckbox(WebElement dualSimCheckbox) {
+        this.dualSimCheckbox = dualSimCheckbox;
+    }
+
+    public WebElement getTouchscreenCheckbox() {
+        return touchscreenCheckbox;
+    }
+
+    public void setTouchscreenCheckbox(WebElement touchscreenCheckbox) {
+        this.touchscreenCheckbox = touchscreenCheckbox;
+    }
+
+    public WebElement getForgCheckbox() {
+        return forgCheckbox;
+    }
+
+    public void setForgCheckbox(WebElement forgCheckbox) {
+        this.forgCheckbox = forgCheckbox;
+    }
+
+    public WebElement getThreegCheckbox() {
+        return threegCheckbox;
+    }
+
+    public void setThreegCheckbox(WebElement threegCheckbox) {
+        this.threegCheckbox = threegCheckbox;
+    }
+
+    public WebElement getWifiCheckbox() {
+        return wifiCheckbox;
+    }
+
+    public void setWifiCheckbox(WebElement wifiCheckbox) {
+        this.wifiCheckbox = wifiCheckbox;
+    }
+
+    public WebElement getExtinderememorieCheckbox() {
+        return extinderememorieCheckbox;
+    }
+
+    public void setExtinderememorieCheckbox(WebElement extinderememorieCheckbox) {
+        this.extinderememorieCheckbox = extinderememorieCheckbox;
+    }
+
+    public WebElement getGpsCheckbox() {
+        return gpsCheckbox;
+    }
+
+    public void setGpsCheckbox(WebElement gpsCheckbox) {
+        this.gpsCheckbox = gpsCheckbox;
+    }
+
+    public WebElement getCameraincorporataCheckbox() {
+        return cameraincorporataCheckbox;
+    }
+
+    public void setCameraincorporataCheckbox(WebElement cameraincorporataCheckbox) {
+        this.cameraincorporataCheckbox = cameraincorporataCheckbox;
+    }
+
+    public WebElement getAgreeCheckbox() {
+        return agreeCheckbox;
+    }
+
+    public void setAgreeCheckbox(WebElement agreeCheckbox) {
```

Comment:
Please remove all setters, because we don't need to set web element.

Reply:
removed